src/scripting/lua_root.cc

```diff
@@ -955,7 +963,7 @@ void LuaDescriptions::do_modify_resource(lua_State* L,
 	Widelands::EditorGameBase& egbase = get_egbase(L);
 	Widelands::Descriptions& descrs = *egbase.mutable_descriptions();
 	Widelands::ResourceDescription& resource_descr =
-	   *descrs.get_mutable_resource_descr(descrs.safe_resource_index(unit_name));
+	   *descrs.get_mutable_resource_descr(descrs.load_resource(unit_name));
```

Comment:
I removed the call in L915 instead. `load_xxx` is safer than `NoteMapObjectDescription` because it also checks that the description has the correct type, and gives a better error message. Performance is the same for both possibilities.

Reply:
ok good to go now.